File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.py
class Solution(object):
    def arraySign(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        res=1
        for n in nums:
            if n==0: 
                return 0
            if n<0:
                res*=-1
        
        return res
        

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.java
class Solution {
    public int arraySign(int[] nums) {
        int res=1;
        for (int num : nums) {
            // System.out.println(num);
            if(num == 0)
                return 0;
            if(num < 0)
                res *= -1;
        }
        return res;
    }
}